projects/cli-files/go-cat/cmd/root.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+)
+
+func Execute() {
+	path, err := checkArg()
+	if err != nil {
+		fmt.Println(err)
+	} else {
```

Comment:
It's common go style to early-return on errors, rather than nesting behaviour in an else; i.e. to write:

```go
if err != nil {
    fmt.Println(err)
    return
}
data, err := os.ReadFile(path)
```

That way your code doesn't all drift to the right from lots of nesting, and it's easier to read what the common/intended case of the code does.